### DISCUSSION TITLE
Placing items in the box at an angle

### DISCUSSION THREAD

**Discussion by joachimdeboer** [dvdoug/BoxPacker#591] [Q&A]
I've been playing around with the boxpacker a bit and I'm impressed, it works very nice and we'll be using this very soon in production.
One question, we sometimes have items that do fit in a box, but then they have to be placed at an angle of let's say 30 degrees. As far as I can see calculations now always are based on items either placed on one of the surfaces, but never in an angle. Is this a possibility?

**Comment by dvdoug** [comment]
Hi @joachimdeboer 

Could you give an example of when you do that - is it e.g. for v. high packing density? (e.g. if the first item was a pyramid, you could lean something against it?)

You're correct that everything in BoxPacker is assumed to be cuboid, and therefore possible orientations are just those of each face (i.e. 90 degree variations)

**Comment by joachimdeboer** [comment]
Thanks! To demonstrate, see this picture:
This item only fits if it's not flat on the ground, these are very light, not breakable items we pack like this.
![image](https://github.com/dvdoug/BoxPacker/assets/5341133/f9ad60df-0708-468e-b1e1-ef28bab17cac)

**Reply by dvdoug** [reply]
Ah OK, I see. Taking advantage of the diagonals. To be honest that kind of feature is not in any kind of mental roadmap I currently have, but it's an interesting one. I'll give it a think
